lib/resolvers/project.js

```diff
@@ -21,7 +21,7 @@ module.exports = ({ models }) => async ({ action, data, id, meta = {} }, transac
       .then(version => {
         if (version.status === 'granted') {
           return Promise.resolve()
-            .then(() => getProfile(meta.changedBy))
+            .then(() => getProfile(changedBy))
```

Comment:
I have manually verified that this is ok for ASRU initiated forks of granted projects.